### PR TITLE
feat: Delete User Profile

### PR DESCRIPTION
**Pull request by monopoint**
image
image
image
image

### CODE REVIEW

**Comment by monopoint** [comment]
Waiting for translation approval: https://mittatb.slack.com/archives/C036F9Q15R8/p1651847598787069

File: src/screens/Profile/DeleteProfile/index.tsx
Comment (gorandalum) [inline src/screens/Profile/DeleteProfile/index.tsx:92]: `Sections.Section` is a wrapper component for wrapping `Section`-items. Rather just set margins with `containerStyle` on `MessageBox` here, and also below on the `unableToDeleteWithTickets`-text.

**Comment by monopoint** [comment]
language aproved by Julia

**Comment by hildeor** [comment]
@monopoint Ser bra ut 👍

**Review by hildeor — APPROVED** [review]

**Pull request merged**
